google/cloud/config/quickstart/quickstart.cc

```diff
@@ -0,0 +1,42 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! [all]
+#include "google/cloud/config/v1/config_client.h"
+#include "google/cloud/project.h"
+#include <iostream>
+
+int main(int argc, char* argv[]) try {
+  if (argc != 3) {
+    std::cerr << "Usage: " << argv[0] << " project-id location-id\n";
+    return 1;
+  }
+
+  namespace config = ::google::cloud::config_v1;
+  auto client = config::ConfigClient(config::MakeConfigConnection());
+
+  std::string const project = argv[1];
+  std::string const location = argv[2];
+  for (auto r : client.ListDeployments("/projects/" + project + "/locations/" +
```

Comment:
The leading slash is not right here.  Resource names start without a slash.  It may be time for us to create a `Location` resource name helper.